strider/fetcher.py

```diff
@@ -59,7 +60,7 @@ def __init__(self, logger, bypass_cache, parameters):
         self.bypass_cache = bypass_cache
         self.parameters = {
             **parameters,
-            "batch_size": parameters.get("batch_size") or 1_000_000,
```

Comment:
How does this effect kg retrieval? Does it have any effect on response time, response rate, or overall process time?